apps/web/src/Pages/Staking/Lockup/Lockup.tsx

```
@@ -139,28 +295,26 @@ function Lockup({
   }
 
   const error = useMemo(() => {
+    if (availableBalance <= txnCost) return false;
     return (
       amount !== "" &&
       (isNaN(Number(amount)) ||
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)